pkgs/development/interpreters/nextflow/default.nix

```
@@ -1,54 +1,96 @@
-{ lib
-, stdenv
-, fetchurl
-, makeWrapper
-, openjdk17
-, wget
-, which
-, gnused
-, gawk
-, coreutils
-, buildFHSEnv
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  openjdk,
+  gradle,
+  wget,
+  which,
+  gnused,
+  gawk,
+  coreutils,
+  bash,
+  versionCheckHook,
+  nixosTests,
 }:
-
-let
-  nextflow =
-stdenv.mkDerivation rec {
+# https://github.com/NixOS/nixpkgs/pull/339197#pullrequestreview-2306770311
+stdenv.mkDerivation (finalAttrs: rec {
```

The whole point of using `finalAttrs` is that you won't have to use a `rec`. See:

https://nix.dev/guides/best-practices#recursive-attribute-set-rec

```suggestion
stdenv.mkDerivation (finalAttrs: {
```